tests/e2e/specs/stories-editor/page.js

```diff
@@ -218,4 +248,102 @@ describe( 'Story Page', () => {
 		const previewPage = await openPreviewPage( editorPage, 'amp-story' );
 		await previewPage.waitForXPath( '//amp-video[contains(@aria-label, "Hello World")]' );
 	} );
+
+	it( 'should pass global story settings to a new story', async () => {
+		const documentPanelSelector = 'button.edit-post-sidebar__panel-tab[data-label="Document"]';
+		const storySettingsPanelSelector = '.amp-story-settings-panel';
+		const storySettingsAdvanceAfterSelector = '.components-panel__body.amp-story-settings-panel .components-select-control__input:nth-of-type(1)';
+		const storySettingsAdvanceAfterDurationSelector = '.components-panel__body.amp-story-settings-panel [aria-label="Time in seconds"]';
+
+		await page.click( documentPanelSelector );
+		await page.waitForSelector( storySettingsPanelSelector );
+
+		await page.click( storySettingsPanelSelector );
+		await page.waitForSelector( storySettingsAdvanceAfterSelector );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, storySettingsAdvanceAfterSelector ) ).toBe( 'time' );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, storySettingsAdvanceAfterDurationSelector ) ).toBe( '2' );
+	} );
+
+	it( 'should save the story settings correctly', async () => {
+		const documentPanelSelector = 'button.edit-post-sidebar__panel-tab[data-label="Document"]';
+		const storySettingsPanelSelector = '.amp-story-settings-panel';
+		const storySettingsAdvanceAfterSelector = '.amp-story-settings-panel .components-select-control__input:nth-of-type(1)';
```

Comment:
This same selector seems to be reused. Could this be a const defined globally?

Reply:
Good idea, added a few `const`s for some of the repeated selectors.